components/header.js

```diff
@@ -274,11 +277,12 @@ function DesktopNavbar() {
               },
             }}
           />
+          <AccountSettingsPopup />
```

Comment:
A discussion we had was about removing the AccountSettings, since they don't actually do anything (at least the e-mail notifications tab). What do you think @AndreiMVP ?